test/crlf.test.js

```diff
@@ -16,20 +16,18 @@ function prettyFactory (opts) {
 
 const logLine = '{"level":30,"time":1522431328992,"msg":"hello world","pid":42,"hostname":"foo"}\n'
 
-test('crlf', (t) => {
-  t.test('uses LF by default', (t) => {
+test('crlf', async (t) => {
```

Comment:
Can you use describe instead?

Reply:
@mcollina I can use whatever you prefer, but please align with @jsumners 🙂
In Pino, he asked me to avoid describe and use test.
So just let me know which way you prefer and I’ll use it.

Reply:
Sorry for the back-and-forth.

describe+test supports only, while the pattern used here does not.

cc @jsumners

Reply:
I think it's part of this? https://github.com/nodejs/node/pull/56664

Reply:
@mcollina Sorry if I ping you.
Is there any update on this one? So we can move forward with these PRs.

Reply:
I'm not going to belabor it. Use `describe`.

Reply:
Done